src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,14 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+	if (employees.length === 0) return 'No one works here!';
+	const empNames = employees.filter((emp) => employeeName === emp.name);
+	if (empNames.length === 0) return `${employeeName} does not work here!`;
+	let string = '';
+	for (let i = 0; i < empNames.length; i++) {
+		i === empNames.length - 1
+			? (string += empNames[i].role)
+			: (string += `${empNames[i].role} `);
+	}
+	return string;
+};
```

Comment:
Interesting take on this problem. Usually, you just return the first role of the employee that has that name, e.g.
```
const getEmployerRole = (employeeName, employees) => {
  return employees.find(employee => employee.name === employeeName).role;
}
```
However, you've set this up to return all roles of an employee that have that name